lib/Controller/SettingController.php

```diff
@@ -118,23 +120,45 @@ public function getUserValue($type) {
 	 * @return JSONResponse
 	 */
 	public function setAppValue($type, $value) {
-		// Apply the change of settings
-		$this->config->setAppValue('facerecognition', $type, $value);
-
-		// Handles special cases when have to do something else according to the change
+		$status = self::STATE_SUCCESS;
 		switch ($type) {
 			case 'sensitivity':
+				$this->config->setAppValue('facerecognition', $type, $value);
 				$this->userManager->callForSeenUsers(function(IUser $user) {
 					$this->config->setUserValue($user->getUID(), 'facerecognition', 'recreate-clusters', 'true');
 				});
 				break;
+			case 'memory-limits':
+				if (is_numeric ($value)) {
+					// Apply prundent limits.
+					if ($value < 1 * 1024 * 1024 * 1024) {
+						$value = 1 * 1024 * 1024 * 1024;
```

Comment:
You asked why send this? The idea is that it can be used as valid value suggestions.

I'll leave it like that, unless they're incongruous, but this led me to think that we can add a parameter 'message', where explicitly explain the error, and show it in the notification. What you think?

Here:
> $messge = "The recommended analysis needs at least 1 Gb of RAM";

Reply:
message is fine to me

Reply:
Done.. Last review and merge it..